src/engine/SCons/Tool/applelink.py

```diff
@@ -40,6 +40,114 @@
 from . import link
 
 
+class AppleLinkInvalidCurrentVersionException(Exception):
+    pass
+
+class AppleLinkInvalidCompatibilityVersionException(Exception):
+    pass
+
+
+def _applelib_versioned_lib_suffix(env, suffix, version):
+    """For suffix='.dylib' and version='0.1.2' it returns '.0.1.2.dylib'"""
+    Verbose = False
+    if Verbose:
+        print("_applelib_versioned_lib_suffix: suffix={:r}".format(suffix))
```

Comment:
shouldn't these be `!r` instead of `:r`?

Reply:
Fixed.